crypto/cryptsetup/BUILD

```diff
@@ -1,5 +1,9 @@
-OPTS+=" --sbindir=/usr/bin \
-        --disable-static \
+OPTS+=" --disable-static \
         --enable-cryptsetup-reencrypt" &&
 
-default_build
+default_build &&
+
+# install hook
+install -D -m644 $SCRIPT_DIRECTORY/encrypt_hook /usr/lib/initcpio/hooks/encrypt &&
```

Comment:
@Florin65 We have no use of initcpio hooks since we don't use a custom mkinitrd. We use dracut for this. These lines can be removed (since you forgot to commit the files anyway).